ee/localserver/dt4a_auth_middleware.go

```diff
@@ -127,6 +139,12 @@ func (d *dt4aAuthMiddleware) Wrap(next http.Handler) http.Handler {
 		bhr := &bufferedHttpResponse{}
 		next.ServeHTTP(bhr, newReq)
 
+		if bhr.code < 200 || bhr.code >= 300 {
+			// hacky sleep here so that it's likely a response form another launcher
+			// on a different port will return with success faster than this failure
+			time.Sleep(1 * time.Second)
```

Comment:
now that we have bumped the rate limit, we could end up with max 200 threads I think 🤔

Reply:
Can we lower to 100ms? That should be enough for the successful path, and I don't know what the extension will experience with the long sleep.

Reply:
done